### DISCUSSION TITLE
Can we have a bittorrent magnet link? And KVM configuration?

### DISCUSSION THREAD

**Discussion by tanc7** [Security-Onion-Solutions/securityonion#6119] [Unsupported Versions] [answered]
I repeatedly tried to download SecurityOnion through the official link and the download fails for some reason. It would be better if I can download it through bittorrent and then calculate the hash checksums.

![Screenshot from 2021-11-05 08-45-17](https://user-images.githubusercontent.com/26513830/140538691-4299f539-2f19-4422-a313-c9ce528bdd7c.png)

I also I intend to install SecurityOnion in Proxmox/KVM once I am finished wiping this used Dell R710. Is there anything I am supposed to know about configuring the management and sniffer interfaces? Like running virsh to create a new virtual NIC prior to running the ISO installation process?

**Comment by dougburks** [comment, marked as answer]
We currently have no plans to offer bittorrent downloads. I just tested our normal ISO image download with no issues. Check your host and network for anything that may be interfering with your ISO download. Alternatively, you could download CentOS 7 or Ubuntu 18.04 and then add our components on top as described here:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos
